remote-wiring-experience/MainPage.xaml.cs

```diff
@@ -348,52 +353,51 @@ private void loadDigitalControls()
                 var text = new TextBlock();
                 text.HorizontalAlignment = HorizontalAlignment.Stretch;
                 text.VerticalAlignment = VerticalAlignment.Center;
-                text.Margin = new Thickness(0, 0, 0, 0);
-                text.Text = "Pin " + i;
+                text.Margin = new Thickness( 0, 0, 0, 0 );
+                text.Text = "Pin " + ( analogPins.Contains( digitalPins[i] ) ? "A" + ( digitalPins[i] - analogOffset ) : "" + digitalPins[i] );
```

Comment:
This looks like it adds both analog and digital pins, but aren't analog pins loaded in the method below? (loadAnalogControls)